theano/tensor/basic.py

```diff
@@ -4057,6 +4057,9 @@ class Tile(Op):
     <http://docs.scipy.org/doc/numpy/reference/generated/numpy.tile.html>`_
```

Comment:
Add a deprecation warning to the docstring of the class. See the function pydotprint_variables() in theano/printing.py for an example of how to do this.